src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1948,7 +1951,17 @@ private static Interval calculateShortFragmentTrim( final GATKRead read, final I
                     read2.setAttribute(ReportType.REPORT_TYPE_ATTRIBUTE_KEY, reportType.attributeValue);
                     rejectedReadsBAMWriter.addRead(read2);
                 }
-            } else { // mates are disjoint
+            } else if (noIgnore) { // mates are disjoint, process both
+                final ReadReport combinedReport = new ReadReport(report1, report2);
+                final ReportType reportType = combinedReport.updateCounts(codonTracker, variationCounts, reference);
+                disjointPairCounts.bumpCount(reportType);
+                if ( reportType.attributeValue != null && rejectedReadsBAMWriter != null ) {
+                    read1.setAttribute(ReportType.REPORT_TYPE_ATTRIBUTE_KEY, reportType.attributeValue);
+                    rejectedReadsBAMWriter.addRead(read1);
```

Comment:
Please delete this line.  Since we're accepting the reads for reporting, they shouldn't be added to the set of rejected reads.

Reply:
Thanks for catching this - removed.